lib/mods_display/fields/nested_related_item.rb

```diff
@@ -21,15 +23,19 @@ def fields
     def to_html(view_context = ApplicationController.renderer)
       helpers = view_context.respond_to?(:simple_format) ? view_context : ApplicationController.new.view_context
 
-      component = ModsDisplay::ListFieldComponent.with_collection(fields, value_transformer: ->(value) { helpers.link_urls_and_email(value.to_s) }, list_html_attributes: { class: 'mods_display_nested_related_items' }, list_item_html_attributes: { class: 'mods_display_nested_related_item open' })
+      component = ModsDisplay::ListFieldComponent.with_collection(fields, value_transformer: lambda { |value|
+                                                                                               helpers.link_urls_and_email(value.to_s)
+                                                                                             }, list_html_attributes: { class: 'mods_display_nested_related_items' }, list_item_html_attributes: { class: 'mods_display_nested_related_item open' })
```

Comment:
huh. This one ended up a little weird. Maybe break it up into a couple of lines??